WalletWasabi/BitcoinCore/Processes/BitcoindRpcProcessBridge.cs

```diff
@@ -20,7 +20,7 @@ public class BitcoindRpcProcessBridge
 		public const string PidFileName = "bitcoin.pid";
 
 		/// <summary>Experimentally found constant.</summary>
-		private readonly TimeSpan ReasonableCoreShutdownTimeout = TimeSpan.FromSeconds(21);
+		private readonly TimeSpan ReasonableCoreShutdownTimeout = TimeSpan.FromSeconds(30);
```

Comment:
I am not sure if this timeout was used somewhere else or not.

Reply:
This value is correct here, I believe. If we encounter any issue elsewhere we will know that 21 seconds may not be enough.